src/test/scala/uk/gov/hmrc/api/specs/BaseSpec.scala

```diff
@@ -19,21 +19,33 @@ package uk.gov.hmrc.api.specs
 import org.scalatest.GivenWhenThen
 import org.scalatest.featurespec.AnyFeatureSpec
 import org.scalatest.matchers.should.Matchers
+import play.api.libs.json.JsValue
+import play.api.libs.ws.StandaloneWSRequest
 import uk.gov.hmrc.api.service.UknwAuthCheckerApiService
-import uk.gov.hmrc.api.specs.payloads.{RequestPayloads, ResponsePayloads}
-import uk.gov.hmrc.api.utils.{JsonReader, TokenReplacement}
 
-import java.time.{LocalDate, LocalDateTime, LocalTime}
+import java.time.{LocalDate, LocalDateTime, LocalTime, ZoneId, ZonedDateTime}
 
-trait BaseSpec
-    extends AnyFeatureSpec
-    with GivenWhenThen
-    with Matchers
-    with RequestPayloads
-    with ResponsePayloads
-    with JsonReader
-    with TokenReplacement {
+trait BaseSpec extends AnyFeatureSpec with GivenWhenThen with Matchers {
 
-  protected val checkerApiService  = new UknwAuthCheckerApiService
-  protected val now: LocalDateTime = LocalDate.now.atTime(LocalTime.MIDNIGHT)
+  protected val checkerApiService       = new UknwAuthCheckerApiService
+  protected val zonedNow: ZonedDateTime = ZonedDateTime.of(LocalDate.now.atTime(LocalTime.MIDNIGHT), ZoneId.of("UTC"))
+  protected val localNow: LocalDate     = LocalDate.now
+
+  def comprehensivelyAssert(
```

Comment:
Could these be methods on an implicit class? That way we can do.

response hasStatusAndBody(OK, Body) for example?